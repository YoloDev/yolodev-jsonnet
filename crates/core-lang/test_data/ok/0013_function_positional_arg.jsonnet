local foo(bar) = bar;

foo(0)
