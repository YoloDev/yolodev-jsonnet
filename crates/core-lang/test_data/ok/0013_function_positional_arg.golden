local foo#2 = function(bar#3 = error "Parameter \'bar\' not bound") bar#3;
foo#2(0.0)
